File: elixir/1101.ex
calculate_sum_of_numbers = fn range ->
  range |> Enum.to_list() |> Enum.sum()
end

generate_sequence_of_numbers = fn range ->
  range
  |> Enum.to_list()
  |> Enum.reduce("", fn value, accumulator -> "#{accumulator} #{value}" end)
  |> String.trim()
end

ranges =
  IO.read(:stdio, :all)
  |> String.trim()
  |> String.split("\n")
  |> Enum.map(fn values ->
    values
    |> String.trim()
    |> String.split(" ")
    |> Enum.map(fn value -> String.to_integer(value) end)
    |> Enum.sort(:asc)
  end)

for range <- ranges, Enum.all?(range, fn value -> value > 0 end) do
  [first, last] = range
  sequence_of_numbers = generate_sequence_of_numbers.(first..last)
  sum = calculate_sum_of_numbers.(first..last)

  IO.puts("#{sequence_of_numbers} Sum=#{sum}")
end

File: javascript/1101.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");

function calculateSumOfNumbers(numbers) {
    return numbers.reduce((accumulator, current) => accumulator + current, 0);
}

function getSequenceOfNumbers(start, end) {
    var sequence = [];

    for (let i = start; i <= end; i++) {
        sequence.push(i);
    }

    return sequence;
}

function main() {
    var numbers = input.split("\n");

    for (var i = 0; i < numbers.length; i++) {
        var [start, end] = numbers[i]
            .split(" ")
            .map((value) => Number(value))
            .sort((a, b) => a - b);

        if (start <= 0 || end <= 0) {
            break;
        }

        var sequenceOfNumbers = getSequenceOfNumbers(start, end);
        var sumOfNumbers = calculateSumOfNumbers(sequenceOfNumbers);

        process.stdout.write(
            `${sequenceOfNumbers.join(" ")} Sum=${sumOfNumbers}\n`
        );
    }
}

main();